src/components/login.jsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import loginAction from "../redux/actions/loginAction";
+import Button from "../views/Button/index";
+import LoginForm from "./LoginForm";
+
+class LoginPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { email: "", password: "" };
+    this.handleChange = this.handleChange.bind(this);
+    this.onLogin = this.onLogin.bind(this);
```

Comment:
I can't recall the last time I saw `binding` statements within a component. I believe if you define `onLogin` and `handleChange` as ES6 statements, you could do away with the above constructor and bind statements, remaining with the state on it's own. Please refer to `https://medium.freecodecamp.org/react-binding-patterns-5-approaches-for-handling-this-92c651b5af56` #5 and refactor all components accordingly. Unless you have a preference to binding statements?